python/tk_multi_workfiles/work_area.py

```diff
@@ -1,16 +1,17 @@
 # Copyright (c) 2015 Shotgun Software Inc.
-# 
```

Comment:
There was a lot of whitespace on non-code lines to I fixed those. Then there were only a few PEP8 errors so I fixed those as well. I will pinpoint where the interesting bits to review are.